docs/wiki/99 - Developers/reCAPTCHA.md

```diff
@@ -0,0 +1,15 @@
+# Enabling reCAPTCHA in LORIS
+
+LORIS can be configured to require [Google reCAPTCHA](https://www.google.com/recaptcha/intro/v3.html) in order to limit spam. These following instructions have been tested for reCAPTCHA v2.
+
+After following Google's instructions for obtaining the your public and private keys, you can
+use them by updating your configuration settings in the `Config` table in the database:
+
+* `reCAPTCHAPrivate` and `reCAPTCHAPublic`. These should be set to the
+corresponding values from your Google reCAPTCHA admin account.
+This can also be configured via the Configuration module by modifying the values found in the "API Keys" heading.
+
+* `CSPAdditionalHeaders` should include "frame-src www.google.com;" in order
```

Comment:
@maltheism These are the instructions you need. Sorry for the confusion

Reply:
@johnsaigle maybe not for this PR but I'm a little confused why the way of going about this is by editing the database to include `frame-src www.google.com;`.  I almost think there should just be a check in the code to see if reCAPTCHAPRIVATE has been added to the config.xml or admin configuration and if so add the `frame-src www.google.com;` to the content security policy. I would think that would be best for having it be hands free for users.

Reply:
Yeah that would probably be a good enhancement.